### ISSUE TITLE
Use of --default-gateway-v6 cause docker daemon crash on startup

### ISSUE THREAD

**Issue by vdavy** [docker/docker#19000]
Here is the command line I use :
> /usr/bin/docker daemon -H fd:// --ipv6 --default-gateway-v6="2a01:e34:ed5f:390::1" --fixed-cidr-v6="2a01:e34:ed5f:390::/124" --graph /home/docker

When I try to restart with **systemctl**, I got a crash and here is the stacktrace in */var/log/syslog*

```
Dec 30 21:05:11 pc-fixe systemd[1]: Started CUPS Scheduler.
Dec 30 21:05:15 pc-fixe systemd[1]: Stopped Docker Application Container Engine.
Dec 30 21:05:15 pc-fixe systemd[1]: Closed Docker Socket for the API.
Dec 30 21:05:15 pc-fixe systemd[1]: Stopping Docker Socket for the API.
Dec 30 21:05:15 pc-fixe systemd[1]: Starting Docker Socket for the API.
Dec 30 21:05:15 pc-fixe systemd[1]: Listening on Docker Socket for the API.
Dec 30 21:05:15 pc-fixe systemd[1]: Starting Docker Application Container Engine...
Dec 30 21:05:15 pc-fixe docker[17560]: time="2015-12-30T21:05:15.264516015+01:00" level=info msg="API listen on /var/run/docker.sock"
Dec 30 21:05:15 pc-fixe docker[17560]: time="2015-12-30T21:05:15.266277086+01:00" level=info msg="[graphdriver] using prior storage driver \"aufs\""
Dec 30 21:05:15 pc-fixe docker[17560]: time="2015-12-30T21:05:15.285846514+01:00" level=info msg="Firewalld running: false"
Dec 30 21:05:15 pc-fixe docker[17560]: time="2015-12-30T21:05:15.294200721+01:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.1/16. Daemon option --bip can be used to set a preferred IP address"
Dec 30 21:05:15 pc-fixe docker[17560]: panic: assignment to entry in nil map
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 1 [running]:
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/docker/daemon.initBridgeDriver(0x7fcabaa9dad0, 0xc2080fc790, 0xc20809d180, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/daemon_unix.go:490 +0xc4d
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/docker/daemon.(*Daemon).initNetworkController(0xc20805ef00, 0xc20809d180, 0x0, 0x0, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/daemon_unix.go:381 +0x69d
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/docker/daemon.NewDaemon(0xc20809d180, 0xc208038428, 0x0, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/daemon.go:770 +0x2149
Dec 30 21:05:15 pc-fixe docker[17560]: main.(*DaemonCli).CmdDaemon(0xc2084f52c0, 0xc20800a020, 0x7, 0x7, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/docker/daemon.go:269 +0x1722
Dec 30 21:05:15 pc-fixe docker[17560]: reflect.callMethod(0xc2084d7f80, 0xc2084ffce8)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/reflect/value.go:605 +0x179
Dec 30 21:05:15 pc-fixe docker[17560]: reflect.methodValueCall(0xc20800a020, 0x7, 0x7, 0x1, 0xc2084d7f80, 0x0, 0x0, 0xc2084d7f80, 0x0, 0x4e6b5f, ...)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/reflect/asm_amd64.s:29 +0x36
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/docker/cli.(*Cli).Run(0xc2084d7ec0, 0xc20800a010, 0x8, 0x8, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/cli/cli.go:89 +0x38e
Dec 30 21:05:15 pc-fixe docker[17560]: main.main()
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/docker/docker.go:65 +0x418
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 5 [syscall]:
Dec 30 21:05:15 pc-fixe docker[17560]: os/signal.loop()
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
Dec 30 21:05:15 pc-fixe docker[17560]: created by os/signal.init·1
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/os/signal/signal_unix.go:27 +0x35
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 17 [syscall, locked to thread]:
Dec 30 21:05:15 pc-fixe docker[17560]: runtime.goexit()
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/runtime/asm_amd64.s:2232 +0x1
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 12 [IO wait]:
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*pollDesc).Wait(0xc208080c30, 0x72, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*pollDesc).WaitRead(0xc208080c30, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*netFD).accept(0xc208080bd0, 0x0, 0x7fcabd784500, 0xc2084f5ba0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/fd_unix.go:419 +0x40b
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*UnixListener).AcceptUnix(0xc208056900, 0xc2084dde10, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/unixsock_posix.go:282 +0x56
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*UnixListener).Accept(0xc208056900, 0x0, 0x0, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/unixsock_posix.go:293 +0x4c
Dec 30 21:05:15 pc-fixe docker[17560]: net/http.(*Server).Serve(0xc20805a8a0, 0x7fcabaa94578, 0xc208056900, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/http/server.go:1728 +0x92
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/docker/api/server.(*HTTPServer).Serve(0xc208056940, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/api/server/server.go:113 +0x4d
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/docker/api/server.func·006(0xc208056940)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/api/server/server.go:86 +0x163
Dec 30 21:05:15 pc-fixe docker[17560]: created by github.com/docker/docker/api/server.(*Server).ServeAPI
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/api/server/server.go:90 +0x144
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 10 [chan receive]:
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/docker/api/server.(*Server).ServeAPI(0xc2080378c0, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/api/server/server.go:94 +0x1b6
Dec 30 21:05:15 pc-fixe docker[17560]: main.func·007()
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/docker/daemon.go:255 +0x3b
Dec 30 21:05:15 pc-fixe docker[17560]: created by main.(*DaemonCli).CmdDaemon
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/docker/daemon.go:261 +0x1571
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 11 [chan receive]:
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/docker/daemon.func·030()
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/debugtrap_unix.go:17 +0x5c
Dec 30 21:05:15 pc-fixe docker[17560]: created by github.com/docker/docker/daemon.setupDumpStackTrap
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/debugtrap_unix.go:20 +0x18e
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 13 [select]:
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/libnetwork.(*controller).watchLoop(0xc2080fc790)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/docker/libnetwork/store.go:398 +0x13f
Dec 30 21:05:15 pc-fixe docker[17560]: created by github.com/docker/libnetwork.(*controller).startWatch
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/docker/libnetwork/store.go:412 +0xf0
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 15 [IO wait]:
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*pollDesc).Wait(0xc208774140, 0x72, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*pollDesc).WaitRead(0xc208774140, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*netFD).readMsg(0xc2087740e0, 0xc20879c1b0, 0x10, 0x10, 0xc20878e420, 0x1000, 0x1000, 0xffffffffffffffff, 0x0, 0x0, ...)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/fd_unix.go:296 +0x54e
Dec 30 21:05:15 pc-fixe docker[17560]: net.(*UnixConn).ReadMsgUnix(0xc208039168, 0xc20879c1b0, 0x10, 0x10, 0xc20878e420, 0x1000, 0x1000, 0x51, 0xc208777f74, 0x4, ...)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/net/unixsock_posix.go:147 +0x167
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/godbus/dbus.(*oobReader).Read(0xc20878e400, 0xc20879c1b0, 0x10, 0x10, 0xc20878e400, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/godbus/dbus/transport_unix.go:21 +0xc5
Dec 30 21:05:15 pc-fixe docker[17560]: io.ReadAtLeast(0x7fcabaa9d588, 0xc20878e400, 0xc20879c1b0, 0x10, 0x10, 0x10, 0x0, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/io/io.go:298 +0xf1
Dec 30 21:05:15 pc-fixe docker[17560]: io.ReadFull(0x7fcabaa9d588, 0xc20878e400, 0xc20879c1b0, 0x10, 0x10, 0x51, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/local/go/src/io/io.go:316 +0x6d
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/godbus/dbus.(*unixTransport).ReadMessage(0xc2087764b0, 0xc208778450, 0x0, 0x0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/godbus/dbus/transport_unix.go:85 +0x1bf
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/godbus/dbus.(*Conn).inWorker(0xc20877a120)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/godbus/dbus/conn.go:248 +0x58
Dec 30 21:05:15 pc-fixe docker[17560]: created by github.com/godbus/dbus.(*Conn).Auth
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/godbus/dbus/auth.go:118 +0xe84
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 16 [chan receive]:
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/godbus/dbus.(*Conn).outWorker(0xc20877a120)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/godbus/dbus/conn.go:370 +0x58
Dec 30 21:05:15 pc-fixe docker[17560]: created by github.com/godbus/dbus.(*Conn).Auth
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/godbus/dbus/auth.go:119 +0xea1
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 18 [chan receive]:
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/libnetwork/iptables.signalHandler()
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/docker/libnetwork/iptables/firewalld.go:92 +0x57
Dec 30 21:05:15 pc-fixe docker[17560]: created by github.com/docker/libnetwork/iptables.FirewalldInit
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/docker/libnetwork/iptables/firewalld.go:48 +0x185
Dec 30 21:05:15 pc-fixe docker[17560]: goroutine 29 [runnable]:
Dec 30 21:05:15 pc-fixe docker[17560]: github.com/docker/libnetwork.(*controller).acceptClientConnections(0xc2080fc790, 0xc208775ce0, 0x63, 0x7fcabaa94578, 0xc2087b02a0)
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/docker/libnetwork/sandbox_externalkey.go:136
Dec 30 21:05:15 pc-fixe docker[17560]: created by github.com/docker/libnetwork.(*controller).startExternalKeyListener
Dec 30 21:05:15 pc-fixe docker[17560]: /usr/src/docker/vendor/src/github.com/docker/libnetwork/sandbox_externalkey.go:132 +0x295
Dec 30 21:05:15 pc-fixe systemd[1]: docker.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Dec 30 21:05:15 pc-fixe systemd[1]: Failed to start Docker Application Container Engine.
Dec 30 21:05:15 pc-fixe systemd[1]: docker.service: Unit entered failed state.
Dec 30 21:05:15 pc-fixe systemd[1]: docker.service: Failed with result 'exit-code'.
```

Any idea ?

**Comment by vdavy** [comment]
```
vincent@pc-fixe:~$ docker version
Client:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 13:20:08 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 13:20:08 UTC 2015
 OS/Arch:      linux/amd64
```
```
vincent@pc-fixe:~$ docker info
Containers: 5
Images: 192
Server Version: 1.9.1
Storage Driver: aufs
 Root Dir: /home/docker/aufs
 Backing Filesystem: extfs
 Dirs: 202
 Dirperm1 Supported: true
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 4.2.0-22-generic
Operating System: Ubuntu 15.10
CPUs: 8
Total Memory: 15.55 GiB
Name: pc-fixe
ID: BYS7:RGHO:EGFH:7VUX:UQUA:3GZV:DANK:DBZH:FKBM:SFDI:3ZGM:O5FF
WARNING: No swap limit support
```

I run docker on a physical PC running kubuntu 15.10. 

## Reproducing the bug :
* Edit the file */etc/systemd/system/docker.service.d/docker-ipv6.conf* to enable IPv6 in docker with this start command line : */usr/bin/docker daemon -H fd:// --ipv6 --default-gateway-v6="2a01:e34:ed5f:390::1" --fixed-cidr-v6="2a01:e34:ed5f:390::/124" --graph /home/docker*
* Reload systemctl daemon with **sudo systemctl daemon-reload**
* Restart docker with **sudo systemctl restart docker**

## Received result :
I got an error from command  **sudo systemctl restart docker** I need to run **systemctl status docker** to get more info and docker is not started.
I read the log file **/var/log/syslog** and got the log entry I posted above.

## Expected result : 
Docker starting and running my containers with specified IPv6 gateway. And no crash

**Comment by thaJeztah** [comment]
ping @mavenugo for libnetwork

**Comment by aboch** [comment]
Looking into it

**Comment by nuved** [comment]
i have this problem too...


root@Ubuntu-1510-wily-64-minimal ~ # docker -d -b=br1 --fixed-cidr-v6=2a01:4f8::2/64 --ipv6 --default-gateway-v6=2a01:4f8::2
Warning: '-d' is deprecated, it will be removed soon. See usage.
WARN[0000] please use 'docker daemon' instead.          
INFO[0000] API listen on /var/run/docker.sock           
INFO[0000] [graphdriver] using prior storage driver "aufs" 
INFO[0000] Firewalld running: false                     
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/docker/docker/daemon.initBridgeDriver(0x7f1a3ab484d8, 0xc20808e580, 0xc208046500, 0x0, 0x0)
        /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/daemon_unix.go:490 +0xc4d
github.com/docker/docker/daemon.(*Daemon).initNetworkController(0xc2080792c0, 0xc208046500, 0x0, 0x0, 0x0, 0x0)
        /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/daemon_unix.go:381 +0x69d
github.com/docker/docker/daemon.NewDaemon(0xc208046500, 0xc20803a410, 0x0, 0x0, 0x0)
        /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/daemon.go:770 +0x2149
main.(*DaemonCli).CmdDaemon(0xc2084f9290, 0xc20800a050, 0x0, 0x0, 0x0, 0x0)
        /usr/src/docker/docker/daemon.go:269 +0x1722
main.handleGlobalDaemonFlag()
        /usr/src/docker/docker/daemon.go:59 +0xe6
main.main()
        /usr/src/docker/docker/docker.go:61 +0x2ac

goroutine 5 [syscall]:
os/signal.loop()
        /usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
        /usr/local/go/src/os/signal/signal_unix.go:27 +0x35

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 12 [chan receive]:
        /usr/src/docker/.gopath/src/github.com/docker/docker/api/server/server.go:90 +0x144

goroutine 10 [chan receive]:
github.com/docker/docker/api/server.(*Server).ServeAPI(0xc2080378c0, 0x0, 0x0)
        /usr/src/docker/.gopath/src/github.com/docker/docker/api/server/server.go:94 +0x1b6
main.func·007()
        /usr/src/docker/docker/daemon.go:255 +0x3b
created by main.(*DaemonCli).CmdDaemon
        /usr/src/docker/docker/daemon.go:261 +0x1571

goroutine 11 [chan receive]:
github.com/docker/docker/daemon.func·030()
        /usr/src/docker/.gopath/src/github.com/docker/docker/daemon/debugtrap_unix.go:17 +0x5c
created by github.com/docker/docker/daemon.setupDumpStackTrap

goroutine 13 [select]:
github.com/docker/libnetwork.(*controller).watchLoop(0xc20808e580)
        /usr/src/docker/vendor/src/github.com/docker/libnetwork/store.go:398 +0x13f
created by github.com/docker/libnetwork.(*controller).startWatch
        /usr/src/docker/vendor/src/github.com/docker/libnetwork/store.go:412 +0xf0

goroutine 15 [IO wait]:
net.(*pollDesc).Wait(0xc208534290, 0x72, 0x0, 0x0)
        /usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
net.(*pollDesc).WaitRead(0xc208534290, 0x0, 0x0)
        /usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
net.(*netFD).readMsg(0xc208534230, 0xc20853bce0, 0x10, 0x10, 0xc208563220, 0x1000, 0x1000, 0xffffffffffffffff, 0x0, 0x0, ...)
        /usr/local/go/src/net/fd_unix.go:296 +0x54e
net.(*UnixConn).ReadMsgUnix(0xc20803a570, 0xc20853bce0, 0x10, 0x10, 0xc208563220, 0x1000, 0x1000, 0x51, 0xc20853baa4, 0x4, ...)
        /usr/local/go/src/net/unixsock_posix.go:147 +0x167
github.com/godbus/dbus.(*oobReader).Read(0xc208563200, 0xc20853bce0, 0x10, 0x10, 0xc208563200, 0x0, 0x0)
        /usr/src/docker/vendor/src/github.com/godbus/dbus/transport_unix.go:21 +0xc5
io.ReadAtLeast(0x7f1a3ab47f90, 0xc208563200, 0xc20853bce0, 0x10, 0x10, 0x10, 0x0, 0x0, 0x0)
        /usr/local/go/src/io/io.go:298 +0xf1
io.ReadFull(0x7f1a3ab47f90, 0xc208563200, 0xc20853bce0, 0x10, 0x10, 0x51, 0x0, 0x0)
        /usr/local/go/src/io/io.go:316 +0x6d
github.com/godbus/dbus.(*unixTransport).ReadMessage(0xc20853a040, 0xc208536330, 0x0, 0x0)
        /usr/src/docker/vendor/src/github.com/godbus/dbus/transport_unix.go:85 +0x1bf
github.com/godbus/dbus.(*Conn).inWorker(0xc20853c5a0)
        /usr/src/docker/vendor/src/github.com/godbus/dbus/conn.go:248 +0x58
created by github.com/godbus/dbus.(*Conn).Auth
        /usr/src/docker/vendor/src/github.com/godbus/dbus/auth.go:118 +0xe84

goroutine 16 [chan receive]:
github.com/godbus/dbus.(*Conn).outWorker(0xc20853c5a0)
        /usr/src/docker/vendor/src/github.com/godbus/dbus/conn.go:370 +0x58
created by github.com/godbus/dbus.(*Conn).Auth
        /usr/src/docker/vendor/src/github.com/godbus/dbus/auth.go:119 +0xea1

goroutine 18 [chan receive]:
github.com/docker/libnetwork/iptables.signalHandler()
        /usr/src/docker/vendor/src/github.com/docker/libnetwork/iptables/firewalld.go:92 +0x57
created by github.com/docker/libnetwork/iptables.FirewalldInit
        /usr/src/docker/vendor/src/github.com/docker/libnetwork/iptables/firewalld.go:48 +0x185

goroutine 29 [runnable]:
github.com/docker/libnetwork.(*controller).acceptClientConnections(0xc20808e580, 0xc208535e30, 0x63, 0x7f1a3ab47650, 0xc2085861a0)
        /usr/src/docker/vendor/src/github.com/docker/libnetwork/sandbox_externalkey.go:136
created by github.com/docker/libnetwork.(*controller).startExternalKeyListener
        /usr/src/docker/vendor/src/github.com/docker/libnetwork/sandbox_externalkey.go:132 +0x295



root@Ubuntu-1510-wily-64-minimal ~ # docker version
Client:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 13:20:08 UTC 2015
 OS/Arch:      linux/amd64
Cannot connect to the Docker daemon. Is the docker daemon running on this host?

**Comment by thaJeztah** [comment]
@nuved this issue is fixed in master and will be in the 1.10 release. Docker 1.9.1 does not yet have this fix
